packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1257,17 +1257,17 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
             this,
             (attachMsg) => this.submit(ContainerMessageType.Attach, attachMsg),
             (id: string, createParam: CreateChildSummarizerNodeParam) => (
-                    summarizeInternal: SummarizeInternalFn,
```

Comment:
I'd suggest splitting formatting PR from functional changes (if possible). Might be a bit late though (I guess it's more feedback for future changes)